.github/workflows/tf-api-rds.yml

```
@@ -0,0 +1,57 @@
+name: tf-api-rds
+run-name: tf-api-rds ${{ inputs.apply || (github.event_name == 'push' && github.ref == 'refs/heads/main') && 'apply' || 'plan' }}
+
+on:
+  push:
+    paths:
+      - .github/workflows/tf-api-rds.yml
+      - terraform/modules/platform/**
+      - terraform/services/api-rds/**
+  workflow_dispatch:
+    inputs:
+      apply:
+        required: false
+        type: boolean
+
+jobs:
+  check-fmt:
+    runs-on: codebuild-cdap-${{github.run_id}}-${{github.run_attempt}}
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./actions/setup-tfenv-terraform
+      - run: terraform fmt -check -diff -recursive terraform/services/api-rds
+
+  plan-apply:
+    needs: check-fmt
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: codebuild-cdap-${{github.run_id}}-${{github.run_attempt}}
+    defaults:
+      run:
+        working-directory: ./terraform/services/api-rds
+    strategy:
+      fail-fast: false
+      matrix:
+        app: [ab2d, bcda, dpc]
+        env: [dev]
+        include:
+          - app: bcda
+            env: test
+          - app: dpc
+            env: test
```

Please exclude `ab2d` for the time being.
```suggestion
        app: [bcda, dpc]
        env: [dev, test]
```

---

Why? We're already managing the ab2d-dev database infra with this service, and plan shows no changes.